### PR TITLE
I seperated html, css and js files and linked them.

### PR DESCRIPTION
**Pull request by KadirOztas**

### CODE REVIEW

File: js/classes.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const classes = [
+    { class: "Fullstack Class", teachers},
```
Comment (KadirOztas) [inline js/classes.js:2]: I don't know what should i write for this "teachers". I need to ask it!

Comment (hi-coders) [reply js/classes.js:2]: Kadir, in code reviews, the developer does not make any comments. You have already done your job by writing your code. Don't add any comments except the answers to my comments/questions.

**Comment by hi-coders** [comment]
Do not use "I ..." format in comments. They should be short and precise.

**Comment by hi-coders** [comment]
the branch name is also tooo general!
Write what you've done in this branvh. Which feature are you implementing etc...

**Review by hi-coders — APPROVED** [review]
You can merge it now

**Pull request merged**